plugins/@grouparoo/sqlite/src/lib/table-import/getColumns.ts

```diff
@@ -0,0 +1,56 @@
+import {
+  GetColumnDefinitionsMethod,
+  ColumnDefinitionMap,
+  FilterOperation,
+  ColumnType,
+} from "@grouparoo/app-templates/dist/source/table";
+
+const filterOperations = [
+  FilterOperation.Equal,
+  FilterOperation.NotEqual,
+  FilterOperation.GreaterThan,
+  FilterOperation.GreaterThanOrEqual,
+  FilterOperation.LessThan,
+  FilterOperation.LessThanOrEqual,
+  FilterOperation.Contain,
+  FilterOperation.NotContain,
+  FilterOperation.In,
+];
+
+const typeMap: { [columnType: string]: ColumnType } = {
+  bigint: "integer",
+  blob: "string",
+  boolean: "boolean",
+  date: "date",
+  datetime: "date",
+  email: "email",
+  float: "float",
+  int: "integer",
+  integer: "integer",
+  num: "float",
+  phoneNumber: "phoneNumber",
+  string: "string",
+  text: "string",
+  url: "url",
+  varchar: "string",
+};
```

Comment:
We could be even fancier here, but for the demo, as long as the columns are named properly (eg: `email`), then the generated config files will be the right types.

Reply:
My thought exactly.